### PR TITLE
Meisha

### PR DESCRIPTION
**Pull request by mam288**
I updated the models with the correct title.

### CODE REVIEW

**Review by balhoff — CHANGES_REQUESTED** [review]
@mam288 can you keep all the filenames the same as they are in the repo?

File: models/AOP_10.ttl [hunk 1]
```diff
@@ -7,7 +7,8 @@
     @base <http://www.w3.org/2002/07/owl#> .
 
     <https://noctua.apps.renci.org/model/AOP_10> a owl:Ontology .
-<https://noctua.apps.renci.org/model/AOP_10> <http://purl.org/dc/elements/1.1/title> "Parkinsonian motor deficits"^^xsd:string .
+<https://noctua.apps.renci.org/model/AOP_10> <http://purl.org/dc/elements/1.1/title> "209    Binding to the picrotoxin site of ionotropic G...
+Name: Title, dtype: object"^^xsd:string .
```
Comment (balhoff) [inline models/AOP_10.ttl]: @meisha I think there is a bug in your title output; see here and other files.

**Review by balhoff — CHANGES_REQUESTED** [review]
There is an issue with the title generation.

**Comment by balhoff** [comment]
> @mam288 can you keep all the filenames the same as they are in the repo?

I renamed the files in this branch to fix.

File: models/AOP_10.ttl (hunk 1 above)
Comment (mam288) [reply models/AOP_10.ttl]: @balhoff What is the bug you're seeing? The issues was that the titles were being assigned to the incorrect AOPs so they probably will be completely different.

Comment (balhoff) [reply models/AOP_10.ttl]: Do you mean for this text to be part of the title? `Name: Title, dtype: object`

Comment (mam288) [reply models/AOP_10.ttl]: Aha! I didn't not mean for that to be included. I'll let you know when I fix that and the file name issue.

**Comment by mam288** [comment]
@balhoff I renamed the files and fixed the titles for the AOPs. Can you load the new files into Noctua for me? Thanks!

**Review by balhoff — APPROVED** [review]

File: models/AOP_1.ttl [hunk 2]
```diff
@@ -7,7 +7,7 @@
     @base <http://www.w3.org/2002/07/owl#> .
 
     <https://noctua.apps.renci.org/model/AOP_1> a owl:Ontology .
-<https://noctua.apps.renci.org/model/AOP_1> <http://purl.org/dc/elements/1.1/title> "Promotion, Hepatocelluar carcinoma"^^xsd:string .
+<https://noctua.apps.renci.org/model/AOP_1> <http://purl.org/dc/elements/1.1/title> "AOP 1"^^xsd:string .
 <https://noctua.apps.renci.org/model/1> <http://geneontology.org/lego/modelstate> "review"^^xsd:string .
```
Comment (balhoff) [inline models/AOP_1.ttl:11]: I just realized there's an issue with the model IDs in the modelstate triples on this branch. I'm going to extract the model titles and add them with a SPARQL update instead.

**Review by balhoff — CHANGES_REQUESTED** [review]

**Comment by balhoff** [comment]
Superseded by d28a8a4f046db9e91de84228c614d6c6a5f17b13.

**Pull request closed without merging**